.github/workflows/deploy.yml

```diff
@@ -123,11 +123,23 @@ jobs:
 
       - name: Update ${{ matrix.environment }} status
         if: ${{ always() }}
-        uses: bobheadxi/deployments@master
+        uses: bobheadxi/deployments@v0.4.3
         with:
           step:   finish
           token:  ${{ secrets.GITHUB_TOKEN }}
           env:    ${{ matrix.environment }}
           ref:    ${{ github.event.inputs.sha }}
           status: ${{ job.status }}
           deployment_id: ${{ steps.deployment.outputs.deployment_id }}
+
+      - name: Alert on Failure
+        if: ${{ failure() && github.ref == 'refs/heads/master' }}
+        uses: rtCamp/action-slack-notify@master
+        env:
+          SLACK_CHANNEL: twd_bat_devops
+          SLACK_COLOR: '#ef5343'
+          SLACK_ICON_EMOJI: ':github-logo:'
+          SLACK_USERNAME: Teacher Training API
+          SLACK_TITLE: Deploy to ${{ matrix.environment }} Failed
+          SLACK_MESSAGE: ':alert: Build failure on ${{ matrix.environment }} :sadparrot:'
```

Comment:
Can we had a channel notification?
Such as https://github.com/DFE-Digital/teaching-vacancies/blob/master/.github/workflows/deploy.yml#L169

Reply:
@saliceti done.